Source/WebKit/UIProcess/API/APIUIClient.h

```diff
@@ -30,6 +30,7 @@
 #include "WKPage.h"
 #include "WebEvent.h"
 #include "WebHitTestResultData.h"
+#include <WebCore/ChromeClient.h>
```

Comment:
This should not need to be included.

Reply:
This is needed.

Reply:
I think @achristensen07 is suggesting that you forward-declare `PointerLockRequestResult`, so you don't need this import:

```suggestion
```
…
```
namespace WebCore {
…
enum class PointerLockRequestResult : uint8_t;
…
}
```